src/main/resources/google_checks.xml

```diff
@@ -48,6 +48,11 @@
 
     <module name="TreeWalker">
         <module name="OuterTypeFilename"/>
+        <module name="WhitespaceAfter">
+            <property name="tokens"
+            value="COMMA, SEMI, TYPECAST, LITERAL_IF, LITERAL_ELSE,
+            LITERAL_WHILE, LITERAL_DO, LITERAL_FOR, DO_WHILE"/>
```

Comment:
How did you identify this list of tokens to add? Issue only talks about commas.

Reply:
Yes, it's true that the issue only talks about commas but the actual google style deals with all these elements and requires them to be followed with a horizontal white-space. Please correct me if I'm wrong. Thanks for the review!
https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace

Reply:
I have not fully looked into the style guide, but if we are doing more than what the issue talks about then the issue needs to be updated to encompass all our changes. In the end, the issue should match what the PR is doing. Issue and issue title is how our users know what we changed as that is what goes on release notes.

Once regression is provided, I will start looking at this in closer details.

Reply:
Since 4.6.2 is numbered, it will be helpful and speed up review if you posted which tokens applies to which number.

Reply:
Alright, I'll try to list them.
1. LITERAL_IF, LITERAL_FOR, LITERAL_WHILE, LITERAL_DO, DO_WHILE
2. LITERAL_ELSE
5. COMMA, SEMI, TYPECAST

Reply:
> 4.6.2.2. LITERAL_ELSE

This seems covered by 4.6.2.3 as 4.6.2.2 is between `}` and `else`.

> TYPECAST

This is the only one I am not sure of because I am not sure of this check, but regression will show what it picks up for it.

I don't see anything else with the tokens added. Other tokens could be added to the check if it supported them.

Reply:
https://shashwat70.github.io/diff2/guava/index.html
Guava shows no violations with new additions.